cumulus/polkadot-parachain/src/chain_spec/bridge_hubs.rs

```diff
@@ -197,8 +176,7 @@ pub mod rococo {
 		parachains_common::rococo::currency::EXISTENTIAL_DEPOSIT;
 
 	/// Specialized `ChainSpec` for the normal parachain runtime.
-	pub type BridgeHubChainSpec = sc_service::GenericChainSpec<(), Extensions>;
-
+	pub type BridgeHubChainSpec = GenericChainSpec;
```

Comment:
```suggestion
```
I think it could be worth removing this type from each mod and just using `GenericChainSpec` directly. That would also mean in `load_config` instead of using `rococo::BridgeHubChainSpec::from_json_bytes` etc. we just use `GenericChainSpec::from_json_bytes` for all live chains.

Reply:
yeah, happy to do that. I was not sure if removal of `BridgeHubChainSpec` is also preferred.

Reply:
done in: 12ad76b